source/includes/_v2_endpoints.md

```diff
@@ -1441,3 +1441,79 @@ the query.
+
+```
+
+```shell
+# Post a link
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=link&url=http%3A%2F%2Fwww.wxyz.com%2Fdecodedc%2Fwarning-for-sitting-governors
+
+# Post an image
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=image&image=http%3A%2F%2Fuser-topic-images.s3.amazonaws.com%2Fbanner%2Fexample_image.jpg&description=Example%20image
+
+# Post a text post
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=text&title=Test%20title&text=Lorem+ipsum+dolor+sit+amet%2C+consectetur+adipiscing+elit%2C+sed+do+eiusmod+tempor+incididunt+ut+labore+et+dolore+magna+aliqua.+Ut+enim+ad+minim+veniam%2C+quis+nostrud+exercitation+ullamco+laboris+nisi+ut+aliquip+ex+ea+commodo+consequat.+Duis+aute+irure+dolor+in+reprehenderit+in+voluptate+velit+esse+cillum+dolore+eu+fugiat+nulla+pariatur.+Excepteur+sint+occaecat+cupidatat+non+proident%2C+sunt+in+culpa+qui+officia+deserunt+mollit+anim+id+est+laborum.
+
+```
+
+This endpoint allows you to post a custom result to a custom source. This result will appear in all topics that have the custom source added to them. 
+
+### HTTP Request
+
+`POST https://api.backstit.ch/v2/organizations/{ORGANIZATION_KEY}/post`
+
+### URL Parameters
+
+| Parameter | Required | Description |
+|---------|:-------:|:-----------|
+| ORGANIZATION_KEY | yes | Your organization’s api key is obtained from the organization dashboard under settings. |
+
+### Query Parameters
+
+These parameters are required regardless of the type of result that is being posted.
+
+| Parameter | Required | Description |
+|---------|:-------:|:-------:|:-----------|
+| feed_name | yes | The name of the custom feed that this result will be posted to. |
+| type | yes | The type of result that is being posted to the feed (text, image, or link). |
+| email | yes | The email address of the user that will be credited for posting this result. |
+
+### Link Parameters
+
+| Parameter | Required | Description |
+|---------|:-------:|:-------:|:-----------|
+| url | yes | The URL of the article that you would like to be scraped for the result. |
+| title | no | What you would like the article to be titled (if none is provided, we will attempt to scrape the article's title from the source). |
```

Comment:
What you would like the article to be titled (if none is provided, backstitch will attempt to scrape the article's title from the source).